sources/functions/radarr

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+_radarr_version() {
+	if radarrv="$(/usr/bin/mono /opt/Radarr/Radarr.exe \? | head -n 1 | grep -Eo "Version(.*)$")"; then
+		[[ "$radarrv" =~ ^(Version 0.2.(.*))$ ]] && echo "mono-v2"
+		[[ "$radarrv" =~ ^(Version 3.0.(.*))$ ]] && echo "mono-v3"
+	elif radarrv="$(/opt/Radarr/Radarr \? | head -n 1 | grep -Eo "Version(.*)$")"; then
+		[[ "$radarrv" =~ ^(Version 3.0.(.*))$ ]] && echo "netcore-v3"
+	else
+		echo "there is no spoon?"
```

Comment:
```suggestion
		echo_error "there is no spoon?"
```
would an `exit 1` also be worthy here?

Reply:
yes, but not an exit. maybe a log or warning